controllers/ocirepository_controller.go

```diff
@@ -681,16 +774,11 @@ func (r *OCIRepositoryReconciler) oidcAuth(ctx context.Context, obj *sourcev1.OC
 
 // craneOptions sets the auth headers, timeout and user agent
 // for all operations against remote container registries.
-func (r *OCIRepositoryReconciler) craneOptions(ctx context.Context, insecure bool) []crane.Option {
```

Comment:
Why have you undone the insecure feature? Please rebase with upstream main instead of merging.

Reply:
thank you, fixed.